### DISCUSSION TITLE
Iframe is blocking clicking at initial page load.

### DISCUSSION THREAD

**Discussion by urakymzhan** [remarkjs/remark#943] [Q&A] [answered]
Hi everyone,
Not sure if i am doing something wrong but since i started using react-markdown whenever i start my project and go to page to click anything on the page i can't click anything. 
After 1 reloading everything starts working fine.
I noticed its because of iframe that react-markdown uses.
Whenever i remove react-markdown package page works seamless.

Any idea where this behavior might come from ?

**Comment by ChristianMurphy** [comment, marked as answer]
Hey @urakymzhan sorry you ran into a spot of trouble.
It likely has to do with something else on the page interacting with `react-markdown`
`react-markdown` itself doesn't add `<iframe>` (unless you explicitly add it in your markdown document)

To help track down the issue, try reducing the number of packages/scripts interacting.
One way to do this is by creating a sandbox, and seeing what is needed to re-create the issue.
Here are some starters which may help:
* https://codesandbox.io/s/github/remarkjs/.github/tree/main/sandbox-templates/react-markdown-with-create-react-app
* https://stackblitz.com/github/remarkjs/.github/tree/main/sandbox-templates/react-markdown-with-create-react-app

Some possible things which could be interfering:
* Another react component, like a modal, could be covering the page
* CSS on the page could be overly broad and targeting the markdown in unexpected ways
* The content you are rendering itself contains an iframe

**Comment by urakymzhan** [comment]
Thanks @ChristianMurphy 
I was able to resolve it.
